Dockerfile

```diff
@@ -1,6 +1,6 @@
 # syntax=docker/dockerfile:1
 
-ARG GO_VERSION=1.19.10
+ARG GO_VERSION=1.20.7
```

Comment:
Was there a specific reason not to update the Dockerfiles to 1.21.0 (now that it's released)?

Reply:
We are taking it slowly. `1.21.x` is very fresh from the oven

Reply:
^ this